kobo-bookmark-extractor.py

```diff
@@ -86,6 +91,8 @@ def handle_data(self, data):
                     parser.feed(book_chapter.read().decode('utf-8'))
         except KeyError:
             pass
+        except NotADirectoryError as ex:
```

Comment:
If I understand the error in your comment correctly, this happens because `book_file` is set to `file:///mnt/onboard/x.mobi/y`, while `file:///mnt/onboard/x.mobi` is a file and not a directory. Am I correct in guessing that this is probably specific to `mobi`-files? Since those aren't supported anyway, the cleaner fix might be to just skip files that don't have the `epub` extension and not even try opening them.

Reply:
Implemented in 9c575af292eab3e3dfd8321b497a34350cddfc00